src/index.js

```diff
@@ -28,6 +53,6 @@ module.exports = async function({
   await page.setContent(html)
   const element = await page.$('body')
   const buffer = await element.screenshot({ path: output, type, omitBackground: transparent, encoding, ...screeshotArgs })
-  await browser.close()
```

Comment:
Could we try [closing the page](https://pptr.dev/#?product=Puppeteer&version=v4.0.1&show=api-pagecloseoptions) here instead, to free up memory?

Reply:
Disclaimer: I don't know if this would work, I expect that you're more familiar with Puppetteer then me 🙈